src/keygen/Avvm.hs

```diff
@@ -130,3 +82,18 @@ genGenesis avvm genCerts holder = GenesisData
         let addr = makeRedeemAddress pk
             adaCoin = unsafeIntegerToCoin $ coinAmount coin
         return (addr, (adaCoin, distr adaCoin))
+
+getHolderId :: Maybe FilePath -> IO StakeholderId
```

Comment:
There used to be a way to specify that no delegation would be done (`noholder`). Is it impossible now? If so, why? (I've no idea about what the actual requirements are, maybe it's really not needed)